hail/src/main/scala/is/hail/expr/ir/GenericTableValue.scala

```diff
@@ -71,6 +71,7 @@ class PartitionIteratorLongReader(
 
   def toJValue: JValue = {
     JObject(
+      "category" -> JString("PartitionIteratorLongReader"),
```

Comment:
not an important change, just makes it a little easier to diagnose what reader is being generated from IR printouts.